File: src/main/java/renderer/constants/CullMode.java
package renderer.constants;

import org.lwjgl.opengl.GL11;

public enum CullMode {
    FRONT(GL11.GL_FRONT), BACK(GL11.GL_BACK);

    public final int glMode;

    CullMode(int glMode) {
        this.glMode = glMode;
    }
}

File: opengl/src/main/kotlin/de/hanno/hpengine/graphics/renderer/constants/CullMode.kt
package de.hanno.hpengine.graphics.renderer.constants

import org.lwjgl.opengl.GL11

val CullMode.glMode: Int get() = when(this) {
    CullMode.FRONT -> GL11.GL_FRONT
    CullMode.BACK -> GL11.GL_BACK
}